assignments/prototypes.js

```diff
@@ -15,14 +15,29 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+function GameObject(p) {
+    this.createdAt = p.createdAt;
+    this.name = p.name;
+    this.demensions = p.dimensions;
+}
+GameObject.prototype.destroy = function () {
+    return `${this.name}  was removed from the game`;
+};
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
-
+function CharacterStats(p) {
+    GameObject.call(this, p);
+    this.isChild = p.isChild;
+    this.healthPoints = p.healthPoints;
```

Comment:
and after inheriting from GameObject, you add two more characteristics (isChild, healthPoints) to CharacterStats